### PR TITLE
fix: Add missing translation for force new password page

### PR DESCRIPTION
**Pull request by ErikCH**
#### Description of changes

Added missing translation for force new password page for React

#### Issue #, if available
#1789 

#### Description of how you validated changes
Added cypress test

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [x] Tests are updated
- [x] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated
- [x] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/react/src/components/Authenticator/ForceNewPassword/ForceNewPassword.tsx [hunk 1]
```diff
@@ -41,11 +41,8 @@ export const ForceNewPassword = ({
           <Heading level={3}>{translate('Change Password')}</Heading>
 
           <FormFields />
-          {error && (
-            <Text className="forceNewPasswordErrorText" variation="error">
-              {error}
-            </Text>
-          )}
+
+          <RemoteErrorMessage />
```
Comment (ErikCH) [inline packages/react/src/components/Authenticator/ForceNewPassword/ForceNewPassword.tsx:45]: This should be using `RemoteErrorMessage` so it looks and behaves like our other error messages in other components.

Comment (wlee221) [reply packages/react/src/components/Authenticator/ForceNewPassword/ForceNewPassword.tsx:44]: Nice and clean!

**Review by wlee221 — APPROVED** [review]
🛥️

**Review by calebpollman — APPROVED** [review]
LGTM 🚢

**Pull request merged**